pages/fellows/BoZheng.md

```diff
@@ -13,7 +13,7 @@ institution: Rice University
 website: https://github.com/coolalexzb
 e-mail: bz30@rice.edu
 mentors:
-  - matthewfeickert (IRIS-HEP, pyhf dev team)
+  - Matthew Feickert (IRIS-HEP, pyhf dev team)
```

Comment:
```suggestion
  - matthewfeickert
```

Would match (and add your institution), but you'd lose the extra text.

Reply:
How about if I add markdown support to the mentor list items; then you can add a link and customize the text as well?

Reply:
Or you can, in this PR:

```diff
diff --git a/_layouts/fellow.html b/_layouts/fellow.html
index 8d891f4..a2a39ed 100644
--- a/_layouts/fellow.html
+++ b/_layouts/fellow.html
@@ -41,7 +41,7 @@ More information: <a href = {{page.proposal}}>My full proposal</a><br>
 {% endif %}
 {% endfor %}
 {% if ok == 0 %}
-{{mentor}}
+{{mentor | markdownify }}
 {%endif%}
 </li>
 {% endfor %}
```

Reply:
@henryiii Let me do that and then change this PR title and body.